ops/op2/dispatch_async.rs

```diff
@@ -144,6 +138,12 @@ pub(crate) fn generate_dispatch_async(
     quote!()
   };
 
+  let with_scope = if generator_state.needs_scope {
```

Comment:
Is this moved on purpose?

Reply:
yes, the order that needs_scope is set changed.